### PR TITLE
Safer alloc

### PR DESCRIPTION
**Pull request by outscale-mgo**

### CODE REVIEW

File: src/switch.c [hunk 1]
```diff
@@ -256,6 +256,8 @@ static int switch_burst(struct pg_brick *brick, enum pg_side from,
 	return forward_bursts(state, source, pkts, errp);
 }
 
+#include "utils/malloc.h"
```
Comment (outscale-toa) [inline src/switch.c]: Can you move this ``` include ... ``` on the top of file please ?

File: tests/vtep/tests.c [hunk 2]
```diff
@@ -811,8 +830,51 @@ static void test_vtep_vnis(int flag)
...
+
+	pg_packets_prepend_udp(pkts, mask, 1000, PG_VTEP_DST_PORT,
+			       1400);
+	pg_packets_prepend_ipv4(pkts, mask, 0x000000EE,
+				0xE70000E7, len, 17);
+	pg_packets_prepend_ether(pkts, mask, &mac1, &mac1,
+				 ETHER_TYPE_IPv4);
+
+	pg_malloc_should_fail = 1;
+	g_assert(pg_brick_burst_to_east(vtep, 0,  pkts, mask, &error) < 0);
```
Comment (outscale-toa) [inline tests/vtep/tests.c]: too many space between  ``` ... 0,  pkts, ... ```

File: tests/vtep/tests.c [hunk 3]
```diff
@@ -811,8 +830,51 @@ static void test_vtep_vnis(int flag)
...
+	pg_packets_prepend_vxlan(pkts, mask, 0);
+
+	pg_packets_prepend_udp(pkts, mask, 1000, PG_VTEP_DST_PORT,
+			       1400);
+	pg_packets_prepend_ipv4(pkts, mask, 0x000000EE,
+				0xE70000E7, len, 17);
+	pg_packets_prepend_ether(pkts, mask, &mac1, &mac1,
+				 ETHER_TYPE_IPv4);
+
+	g_assert(!pg_brick_burst_to_east(vtep, 0,  pkts, mask, &error));
```
Comment (outscale-toa) [inline tests/vtep/tests.c]: too many space between  ``` ... 0,  pkts, ... ```

**Comment by outscale-mgo** [comment]
merge with https://github.com/outscale/packetgraph/pull/417

**Pull request closed without merging**